lib/stream.js

```diff
@@ -12,3 +14,38 @@ Stream.PassThrough = require('_stream_passthrough');
 
 // Backwards-compat with node 0.4.x
 Stream.Stream = Stream;
+
+// Internal utilities
+try {
+  Stream._isUint8Array = process.binding('util').isUint8Array;
+} catch (e) {
+  // This throws for Node < 4.2.0 because there’s no util binding and
+  // returns undefined for Node < 7.4.0.
+}
+
+if (!Stream._isUint8Array) {
+  Stream._isUint8Array = function _isUint8Array(obj) {
+    return Object.prototype.toString.call(obj) === '[object Uint8Array]';
+  };
+}
+
+const version = process.version.substr(1).split('.');
+if (version[0] <= 0 && version[1] < 12) {
```

Comment:
Because `readable-stream` still supports them and removing that support would at least be a breaking change…

> When would the major version be less than zero?

Never, I can update this to `===`